sdk/tables/azure-data-tables/tests/test_table_entity.py

```diff
@@ -320,11 +320,12 @@ def test_query_user_filter(self, tables_storage_account_name, tables_primary_sto
             entity = self._insert_random_entity()
 
             # Act
-            # resp = self.table.create_item(entity)
-            resp = self.table.query_entities(filter="married eq @my_param", parameters={'my_param': 'True'})
+            entities = self.table.query_entities(filter="married eq @my_param", parameters={'my_param': 'True'})
+
+            assert entities is not None
+            for e in entities:
+                self._assert_default_entity(e)
```

Comment:
We should also assert the filter results. For example - we insert 2 entities where `my_param` is True and False, then the query results - when fully listed - should only have a single entity returned.